backend/inspirehep/records/marshmallow/base.py

```diff
@@ -89,3 +89,21 @@ class MetadataSchema(EnvelopeSchema):
         metadata = fields.Nested(schema, dump_only=True)
 
     return MetadataSchema
+
+
+class RequiredFieldsSchema(Schema):
```

Comment:
IMHO this is too complex, each just a check on the specific serializer instead on ``post_dump`` or even better https://github.com/inspirehep/inspirehep/pull/498/files#diff-19d3d162ce2a08a6469fc1ae8afae85fR112